shared_model/backend/protobuf/queries/proto_query.hpp

```diff
@@ -49,13 +49,13 @@ auto load_query(Archive &&ar) {
 
 namespace shared_model {
   namespace proto {
-    class Query final : public CopyableProto<interface::Query,
+    class Query FINAL : public CopyableProto<interface::Query,
                                              iroha::protocol::Query,
                                              Query> {
      private:
       /// polymorphic wrapper type shortcut
-      template <typename... Value>
-      using wrap = boost::variant<detail::PolymorphicWrapper<Value>...>;
+      template <typename Value>
+      using wrap = detail::PolymorphicWrapper<Value>;
```

Comment:
So sad when variadic templates are missing in SWIG.

Reply:
The reality is even more fun - Swig supports both variadic templates and using typedefs but it can't parse them in a single statement.